client/src/replay_history.json

```diff
@@ -0,0 +1,112 @@
+{
+    "HistoryHistories": [
+        {
+            "user-1": [
+                {
+                    "Id": "1",
```

Comment:
consider adding this field to Bitempura. However, this starts to further bake in assumptions about implementation of the bitemporal records. We are further and further leaning into describing a very specific implementation style